model/datasetPage/model.go

```diff
@@ -0,0 +1,48 @@
+package datasetPage
+
+import "github.com/ONSdigital/dp-frontend-models/model"
+
+//Page contains data re-used for each page type a Data struct for data specific to the page type
+type Page struct {
+	model.Page
+	DatasetPage DatasetPage `json:"data"`
+	model.ContactDetails
+}
+
+//Dataset has the file and title information for an individual dataset
+type DatasetPage struct {
+	Versions            []Version           `json:"versions"`
+	SupplementaryFiles  []SupplementaryFile `json:"supplementary_files"`
+	Downloads           []Download          `json:"downloads"`
+	IsNationalStatistic bool                `json:"national_statistic"`
+	ReleaseDate         string              `json:"releaseDate"`
```

Comment:
I think our json naming convention requires these to be `release_date`, `next_release`, `dataset_id`etc.

Reply:
`snake_case`